### PR TITLE
Make the header and the footer responsive

### LINKED ISSUE #46: The header and the footer are not responsive
**Describe the bug**
- In mobile view header and the footer in not responsive

**To Reproduce**
1. Go to https://sefglobal.org/academix/
2. Go to the mobile view
3. See error

**Expected behavior**
- Make it responsive

**Screenshots**
image

image

**Desktop (please complete the following information):**
 - OS: macOS 10.15.5
 - Browser: chrome

**Are you working on this issue?**
Yes

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #46 

## Goals

- Make the header and the footer responsive in mobile view

## Approach

- Add SEF logo as a menu item
- Remove menu item from the footer
image
image

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: app/src/components/App/index.tsx [hunk 1]
```diff
@@ -72,19 +77,11 @@ class App extends React.Component<AppProps> {
...
-            >
-              <Menu.Item key="1">
-                © 2020{' '}
-                <a href="https://sefglobal.org/">
-                  {' '}
-                  Sustainable Education Foundation - SEF
-                </a>
-              </Menu.Item>
-            </Menu>
+            © 2020{' '}
```
Comment (jayasanka-sack) [inline app/src/components/App/index.tsx:80]: let's remove this `{' '}`

File: app/src/components/App/index.tsx [hunk 2]
```diff
@@ -72,19 +77,11 @@ class App extends React.Component<AppProps> {
...
+            <a href="https://sefglobal.org/">
+              {' '}
```
Comment (jayasanka-sack) [inline app/src/components/App/index.tsx:82]: Let's add margin instead of this white space

File: app/src/components/App/index.tsx (hunk 1 above)
Comment (anjula-sack) [reply app/src/components/App/index.tsx:80]: ok

File: app/src/components/App/index.tsx (hunk 2 above)
Comment (anjula-sack) [reply app/src/components/App/index.tsx]: Done

**Pull request merged**